README.md

```diff
@@ -23,61 +23,23 @@ This repository contains two crates:
   Workers](https://workers.cloudflare.com/). This crate also implements the
   various HTTP endpoints defined in the DAP spec.
 
+* `daphne_worker_test` -- Defines a deployment of Daphne-Worker for testing
+  changes locally. It also implements integration tests for Daphne and
+  Daphne-Worker and interop tests with
+  [Janus](https://github.com/divviup/janus).
+
 ## Testing
 
-> NOTE These instructions will be changed as part of a planned refactor of
-> Daphne-Worker and the tests. See issue
-> [#1](https://github.com/cloudflare/daphne/issues/1) for details.
+The `daphne` crate relies on unit tests. The `daphne_worker` crate relies mostly
+on integration tests implemented in `daphne_worker_test`. See
+[daphne_worker_test/README.md](daphne_worker_test/README.md) for details.
 
-The `daphne` crate relies on unit tests. The `daphne_worker` crate contains a
-number of integration and end-to-end tests that use
-[miniflare](https://miniflare.dev/) to mock the Cloudflare Workers platform.
-These can via docker-compose:
+Integration tests can be run via docker-compose:
 
 ```
 docker-compose up --build --abort-on-container-exit --exit-code-from test
 ```
```

Comment:
Reviewer: Please check if this runs.

Reply:
this runs ok locally, it's takes long time though.

Reply:
Filed https://github.com/cloudflare/daphne/issues/31 to address this.